### ISSUE TITLE
import name error

### ISSUE THREAD

**Issue by darouwan** [dlitz/pycrypto#151]
In the README document, it indicates <pre>from Crypto.Cipher import AES</pre>, which cannot work with error info 'ImportError: No module named 'Crypto''. 

It should be <pre>from crypto.Cipher import AES</pre> . Import in Python is case-sensitive. I am not sure if it is related to my OS or python version.

My environment is Windows 10 64bit, and Python 3.4

**Comment by Jurph** [comment]
I'm seeing the same problem.  My environment is PyCharm for Win7 and "import pycrypto", "import crypto" and "import Crypto" *all* fail.

**Comment by darouwan** [comment]
@Jurph  @dlitz  Exactly. I have fixed it on my branch, but in meet a problem during pulling request, that remind me compiling error.  It should not occur since I just change some words in a normal text file.

**Comment by kamakazikamikaze** [comment]
I believe it's resolved just by renaming the library folder.
![capture](https://cloud.githubusercontent.com/assets/8862823/10595356/5f45498e-7694-11e5-892e-93bba06af19d.PNG)

If your library has installed it as `crypto` instead of `Crypto`, simply capitalizing it seems to fix the issue. Exit any IDE you have open, make the change, then reopen and verify if/that it works

**Comment by darouwan** [comment]
@kamakazikamikaze  thanks. I just want to fix this problem permanently by changing the import name, and assuming the source code is right.
